### ISSUE TITLE
screen reader cannot read the file properties correctly (L1)

### ISSUE THREAD

**Issue by JillieBeanSim** [zowe/zowe-explorer-vscode#4024]
**Describe the bug**

When selecting a file or folder under USS to edit its attributes, the screen reader does not correctly read the file properties or clearly indicate how to update the file attributes

**To Reproduce**

**Expected behavior**

**Screenshots**

**Desktop (please complete the following information):**

- OS:
- Zowe Explorer Version:
- (Optional) Zowe CLI Version:
- (Optional) Do you have secure credentials enabled?
- (Optional) Are you using a custom credential manager?

**Additional context**
